graviti/openapi/sheet.py

```diff
@@ -68,14 +71,13 @@ def create_sheet(
         ...     "MNIST",
         ...     draft_number = 1,
         ...     name = "val",
-        ...     schema = "{\"imports\": [{\"repo\":
-        ...         \"https://github.com/Project-OpenBytes/standard@main\", \"types\":
-        ...         [{\"name\": \"label.Category\"}, {\"name\": \"tensor.Image\"}]}],
-        ...         \"type\": \"record\", \"fields\": [{\"name\": \"image\", \"type\":
-        ...         \"tensor.Image\", \"dtype\": \"uint8\", \"shape\": [28, 28]}, {\"name\":
-        ...         \"category\", \"type\": \"label.Category\", \"categories\": [0, 1, 2, 3,
-        ...         4, 5, 6, 7, 8, 9]}]}",
-        ...     _arrow_schema = ""
+        ...     schema = '{"imports": [{"repo": "https://github.com/Project-OpenBytes/standard@\
```

Comment:
shall we update the example in another PR?

Reply:
ok